internal/controller/pdbwatcher_controller.go

```diff
@@ -113,7 +112,7 @@ func (r *PDBWatcherReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 		return ctrl.Result{}, err // Error fetching Deployment
 	}
-
+	logger.Info(fmt.Sprintf("pdbwatcher is at generation %d, Target generation is at %d", pdbWatcher.Status.TargetGeneration, target.Obj().GetGeneration()))
```

Comment:
should we only log this when the generations are differnt so we don't overlog.

Reply:
can remove this... just added it for debugging